.github/workflows/check-pr-title.yml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Enforce Conventional Commits on PR titles.
+# This helps our automation systems infer things about the change
+# to apply labels and categorize release notes.
+name: PR Title
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - edited
+      - synchronize
```

Comment:
If we choose to land this, let's add a `permissions` stanza.

See dicussion [here](https://github.com/googleapis/repo-automation-bots/issues/3733#issuecomment-1142519301) as to why switching to always linting the PR title is potentially not the behavior we want.